auth-web/src/components/auth/staff/DissolutionSchedule.vue

```diff
@@ -0,0 +1,296 @@
+<template>
+  <div id="summary-define-company">
+    <section>
+      <article class="section-container px-6 py-8">
+        <!-- Dissolution Batch Size -->
+        <template v-if="isEdit">
+          <v-row no-gutters>
+            <v-col
+              cols="12"
+              sm="3"
```

Comment:
What does this look like on a narrow screen? I don't we don't try hard to be responsive but you might be able to copy some props from other UIs so that these columns stack nicely, with nice whitespace.

Reply:
I tried to test and make it look good as much as possible by playing with the v-cols throughout it all. This is what I have as of right now on a narrow screen:
![narrow screen](https://github.com/bcgov/sbc-auth/assets/122301442/b16491c3-8ec8-4511-bc83-717a5218eccb)

I think it looks good. Thoughts?

Reply:
LGTM